hw-8/index.js

```diff
@@ -0,0 +1,89 @@
+// eslint-disable-next-line import/extensions
+import galleryItems from './gallery-items.js';
+
+// add images
+const ulGallery = document.querySelector('ul.gallery.js-gallery');
```

Comment:
Тут не обов'яково використовувати таку довгу вибірку, краще залишити тільки один клас
```suggestion
const ulGallery = document.querySelector('.js-gallery');
```